.coverage.rc

```diff
@@ -0,0 +1,35 @@
+# .coveragerc to control coverage.py
```

Comment:
```
Coverage.py warning: Couldn't read data from '/home/stenegru/csc/beacon-network/.coverage.rc': CoverageException: Doesn't seem to be a coverage.py data file
```
renaming maybe to `.coveragerc` will help

Reply:
renamed